container/push.bzl

```diff
@@ -58,6 +58,13 @@ def _impl(ctx):
     config_arg = "--config=%s" % _get_runfile_path(ctx, image["config"])
     manifest_arg = "--manifest=%s" % _get_runfile_path(ctx, image["manifest"])
 
+    optional_args = ""
```

Comment:
add this optional_args to line 89 below so you dont need to modify the template

Reply:
Done